### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by PrajwalM2212**
Detailed step by step guide for setting up dev environment and for sending pull requests

### CODE REVIEW

**Comment by PrajwalM2212** [comment]
I have sent the PR to master branch as it has to be easily accessible to GCI students.

**Comment by chfw** [comment]
Got you! Will merge it later.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Steps for creating a Pull Request
+1. Checkout to the develop branch `git checkout develop`
+3. Start a new branch with a suitable name `git checkout -b branch_name`
+4. Develop a new feature or solve an existing issue 
+5. Add the changed files `git add file_name`
+6. Commit with a suitable message `git commit -m " Changes made "`
+7. Push `git push origin branch_name`
+8. Go to the Github Repository and create a pull request to the develop branch
+
```
Comment (jayvdb) [inline CONTRIBUTING.md]: trailing blank line.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Fork [moban](https://github.com/moremoban/moban) by clicking "Fork".
+
+## Using virtualenv and Remote Configuration
+1. `pip3 install virtualenv`
+2. Move to the location where you want to setup moban and type `mkdir moban` in the terminal
+3. `cd moban`
+3. Type `virtualenv venv`
+4. `source venv/bin/activate`
+5. `git clone git clone https://github.com/YOUR_USERNAME/moban.git`
+6. `cd moban`
```
Comment (Man-Jain) [inline CONTRIBUTING.md]: git clone typed twice

Comment (PrajwalM2212) [reply CONTRIBUTING.md]: Done . Thanks for the review 😊

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+## Using virtualenv and Remote Configuration
+1. `pip3 install virtualenv`
+2. Move to the location where you want to setup moban and type `mkdir moban` in the terminal
+3. `cd moban`
+3. Type `virtualenv venv`
+4. `source venv/bin/activate`
+5. `git clone https://github.com/YOUR_USERNAME/moban.git`
+6. `cd moban`
+7. `pip3 install -e .`
+7. `git remote add origin https://github.com/YOUR_USERNAME/moban.git`
```
Comment (Man-Jain) [inline CONTRIBUTING.md]: when we clone a repo from our fork then the origin is already set so this would most probably give an error.

Comment (Man-Jain) [reply CONTRIBUTING.md]: yupp

**Pull request merged**